ibis/backends/flink/__init__.py

```diff
@@ -3,13 +3,15 @@
 from functools import lru_cache
 from typing import TYPE_CHECKING, Any
 
+import pandas as pd
```

Comment:
I think elsewhere (like in `create_table`) we're gating this import inside of the method that needs to the `isinstance` check.  This should probably get moved down inside of `insert`